aflr/helper_classes.py

```diff
@@ -1,3 +1,6 @@
+import os
+from posixpath import basename
```

Comment:
remove basename import (not used) and the newline after it

Reply:
done